ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```
@@ -586,6 +586,37 @@ tests:
     test:
     - ref: fips-check-optional-operator-images-scan
     workflow: cucushift-installer-rehearse-aws-ipi-proxy
+- as: aws-ipi-proxy-fips-f360-compliance-operator
+  cron: 25 19 16 2 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FILTERS_ADDITIONAL: ""
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~HyperShiftMGMT&;~Disruptive&
+      TEST_PARALLEL: "3"
+      TEST_SCENARIOS: Compliance_Operator
+    test:
+    - ref: openshift-extended-test
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy
+- as: aws-ipi-proxy-fips-f360-compliance-operator-destru
```

```suggestion
- as: aws-ipi-proxy-fips-f360-compliance-destructive
```

so that we can keep `destructive` keyword here ?

---

I would like to.
However, the make update will report similar error as below:
time="2024-10-22T00:50:59Z" level=error msg="Failed to load CI Operator configuration" error="invalid ci-operator config: invalid configuration: tests[11].as: 65 characters long, maximum length is 61" source-file=/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable.yaml

---

```
$ a=aws-ipi-proxy-fips-f360-compliance-destructive
$ echo ${#a}
46
```
The job name has only 46 characters, does not exceed the limit (61).

And from your pasted error log, it is from `openshift-openshift-tests-private-release-4.13__amd64-stable.yaml`, I did not see any update for the yaml in this PR.

---

suffered from similar logs. Not exactly the same. 
Sure I will keep the suffix as "compliance-destructive"